ArcadiaTeamsBot/Startup.cs

```diff
@@ -1,22 +1,28 @@
+using ArcadiaTeamsBot.CQRS.Queries;

```

Comment:
No, should be reverted. It is our code-style - to have usings inside a namespace.

Reply:
Ok. I will follow the code-style